source/Engine.h

```diff
@@ -188,6 +188,8 @@ class Engine {
 	// Viewport position and velocity.
 	Point center;
 	Point centerVelocity;
+	Point offset;
+	double offsetMultiplier = 0.0;
```

Comment:
Here as well no need to write 0.0 as 0. is the same

Reply:
done